client/src/Component/NavbarComponent.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Navbar, Nav, Container } from "react-bootstrap";
+
+
+const NavbarComponent = () => {
+	return (
+		<Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+		<Container>
+		<Navbar.Brand as={Link} to="/">CYF | Bike for Refuges SMS</Navbar.Brand>
```

Comment:
```suggestion
		<Navbar.Brand as={Link} to="/">Bike for Refugees SMS</Navbar.Brand>
```
CYF are creating this app but we aren't naming the app after ourselves so we can remove `CYF` from the name

Reply:
Ok, that's understandable.